### ISSUE TITLE
Failed to read the 'cssRules' property from 'CSSStyleSheet': Cannot access rules after upgrading Chrome

### ISSUE THREAD

**Issue by sonlexqt** [practicalmeteor/mocha#101]
I'm using `practicalmeteor:mocha` to run integration tests. The command is as follow:
`meteor test --full-app --driver-package practicalmeteor:mocha --settings .settings/localhost.json --port 5000`. The tests ran well before. However, after upgrading to Chrome version 65.0.3325.162 (64-bit), when I run the tests and access `localhost:5000` to see the tests result, I got this error:
```
Error: Failed to read the 'cssRules' property from 'CSSStyleSheet': Cannot access rules
Error: Failed to read the 'cssRules' property from 'CSSStyleSheet': Cannot access rules
    at http://localhost:5000/packages/tmeasday_test-reporter-helpers.js?hash=332cac4e0d62ca9ab43d6bead1c1f1b85ee1fdbf:71:32
    at Function._.each._.forEach (http://localhost:5000/packages/underscore.js?hash=cde485f60699ff9aced3305f70189e39c665183c:152:22)
    at hideOtherCSS (http://localhost:5000/packages/tmeasday_test-reporter-helpers.js?hash=332cac4e0d62ca9ab43d6bead1c1f1b85ee1fdbf:70:5)
    at Object.runTests (http://localhost:5000/packages/practicalmeteor_mocha.js?hash=75723553f03753058884ffd2be61dfe6bee0e128:164:3)
    at http://localhost:5000/packages/meteor.js?hash=6d285d84547b3dad9717a7c89c664b61b45ea3d8:1040:25
    at maybeReady (http://localhost:5000/packages/meteor.js?hash=6d285d84547b3dad9717a7c89c664b61b45ea3d8:821:28)
    at HTMLDocument.loadingCompleted (http://localhost:5000/packages/meteor.js?hash=6d285d84547b3dad9717a7c89c664b61b45ea3d8:833:5)
```

Have anyone encounter this problem and how to resolve it ?

**Comment by quintstoffers** [comment]
We recently ran into what I assume is the same error. In our case this was caused by an external style sheet (FontAwesome, to be precise). You can find out which sheets are causing the issue by running the following snippet in the console:
```
Object.keys(document.styleSheets).forEach(function (sheetIndex) {
    const sheet = document.styleSheets[sheetIndex];

    try {
        sheet.rules
    } catch (e) {
        console.log("Failure at sheet:", sheet)
    }
});
```

We worked around the issue by not injecting the FontAwesome link when testing (for instance by using `Meteor.isPackageTest`), which is less than ideal but works nonetheless.

**Comment by Jayki-ZX** [comment]
@quintstoffers Hi, I tried copying the snippet on chrome´s console, but didn´t worked for me :(,  It printed out:

 `Failure at sheet: CSSStyleSheet {ownerRule: null, type: "text/css", href: "http://fonts.googleapis.com/css?family=Nunito+Sans:200,300,400,400i,600,700", ownerNode: link, parentStyleSheet: null, …}
VM425:7 Failure at sheet: CSSStyleSheet {ownerRule: null, type: "text/css", href: "http://fonts.googleapis.com/css?family=Roboto:100,300,400,500,700", ownerNode: link, parentStyleSheet: null, …}
VM425:7 Failure at sheet: CSSStyleSheet {ownerRule: null, type: "text/css", href: "http://maxcdn.bootstrapcdn.com/font-awesome/4.6.3/css/font-awesome.min.css", ownerNode: link, parentStyleSheet: null, …}
undefined`


And still not showing up the test client. Did you do something else than running that code?
Can you give me instructions of how to do the last part? 

> We worked around the issue by not injecting the FontAwesome link when testing (for instance by using Meteor.isPackageTest), which is less than ideal but works nonetheless.

**Comment by sonlexqt** [comment]
Hi, I was able to resolve my issue. First, I list out `document.styleSheets` and one of them doesn't have the `cssRules` property. This one comes from the Google Fonts imported in the `<link>` tag. I switched to use CSS `@import` instead and the issue is resolved.
I think there will be a better solution, because [we better to not use @import](https://stackoverflow.com/questions/12316501/including-google-web-fonts-link-or-import/12380004#12380004)

**Comment by Jayki-ZX** [comment]
I have 3 fonts giving me trouble, I tried writing the `@import` statements on my project css file and changing the fonts but It seems like im editing the wrong files, where did you add the import statements? 

Update: Never mind, your fix worked for me too!, although I had to delete `natestrauser:font-awesome` via `meteor remove natestrauser:font-awesome`. Now my projects/test client is working fine, anyway I re-added fontawesome via `meteor add fontawesome:fontawesome` to cover the cases where its needed.

Thanks 👍 !
